src/hotspot/cpu/riscv/templateTable_riscv.cpp

```diff
@@ -209,7 +209,7 @@ void TemplateTable::patch_bytecode(Bytecodes::Code bc, Register bc_reg,
   Label L_okay;
   __ load_unsigned_byte(temp_reg, at_bcp(0));
   __ beq(temp_reg, bc_reg, L_okay);
-  __ addi(temp_reg, temp_reg, -(int) Bytecodes::java_code(bc));
+  __ sub(temp_reg, temp_reg, (int)Bytecodes::java_code(bc));
```

Comment:
ditto

Reply:
Make sense. I have fixes all of them. Thanks.